apps/clinical/src/components/forms/immunization/__tests__/__mocks__/immunizationMocks.ts

```
@@ -242,8 +238,40 @@ export const mockCovid19VaccineDrug: Medication = {
   code: { coding: [{ code: 'covid-19' }] },
 };
 
+export const mockInfluenzaVaccineDrug: Medication = {
+  resourceType: 'Medication',
+  id: 'flu-drug-uuid',
+  extension: [
+    {
+      url: MEDICINE_EXTENSION_URL,
+      extension: [
+        { url: MEDICINE_DRUG_NAME_EXTENSION_URL, valueString: 'Influenza Drug' },
```

ESLint

[error] Replace `·url:·MEDICINE_DRUG_NAME_EXTENSION_URL,·valueString:·'Influenza·Drug'` with `⏎··········url:·MEDICINE_DRUG_NAME_EXTENSION_URL,⏎··········valueString:·'Influenza·Drug',⏎·······`

(prettier/prettier)